openvm/src/utils.rs

```diff
@@ -9,27 +9,21 @@ use openvm_stark_backend::{
         SymbolicConstraints,
     },
     interaction::Interaction,
-    p3_field::PrimeField32,
+    p3_field::{FieldAlgebra, PrimeField32},
 };
 use powdr_ast::analyzed::{
     AlgebraicBinaryOperation, AlgebraicBinaryOperator, AlgebraicExpression, AlgebraicReference,
     AlgebraicUnaryOperation, AlgebraicUnaryOperator, PolyID, PolynomialType,
 };
-use powdr_number::{BabyBearField, FieldElement, LargeInt};
+use powdr_number::{BabyBearField, FieldElement, OpenVmField as OpenVmFieldTrait};
 
-pub fn to_powdr_field<F: PrimeField32, P: FieldElement>(f: F) -> P {
-    f.as_canonical_u32().into()
-}
-
-pub fn to_ovm_field<F: PrimeField32, P: FieldElement>(f: P) -> F {
-    F::from_canonical_u32(f.to_integer().try_into_u32().unwrap())
-}
+pub(crate) type OvmField<PowdrField> = <PowdrField as OpenVmFieldTrait>::Field;
```

Comment:
Bit confusing to have `OpenVmField` (sometimes imported as `OpenVmFieldTrait`) and `OvmField`.

I'd propose: `OpenVmField` -> `IntoOpenVm`, `OvmField` -> `OpenVmField`.

Reply:
Done